### PR TITLE
Issues

### PR DESCRIPTION
**Pull request by teatimes**
Fixed some issues.

### CODE REVIEW

File: src/client/components/FeedbackMenu.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ const style = {
   minWidth: '105px'
 }
 
+var MIN = 6000*60
```
Comment (odgaard) [inline src/client/components/FeedbackMenu.js]: Expect this is supposed to be 1000*60?

File: src/client/components/LectureTable.js [hunk 2]
```diff
@@ -51,6 +51,7 @@ export default class LectureTable extends Component {
     componentDidMount() {
         /* This method is called after first render */
         this.getAvailableLectures()
+        console.log(new Date())
```
Comment (odgaard) [inline src/client/components/LectureTable.js]: What is this used for?

File: src/client/components/ChatBox.js [hunk 3]
```diff
@@ -52,28 +53,50 @@ export default class ChatBox extends Component {
...
+    }
+    
   }
   // Listen and update field dynamically when something is written
   changeHandler (e) {
     this.setState({ text: e.target.value })
     console.log('[ChatBox] changeHandler')
   }
 
+  validateMessage (msg) {
```
Comment (odgaard) [inline src/client/components/ChatBox.js]: The error message overflows when the message box has 2 lines or more of text:
image

**Review by odgaard — CHANGES_REQUESTED** [review]
All tests pass, everything is functional on the website.

File: src/client/components/FeedbackMenu.js (hunk 1 above)
Comment (teatimes) [reply src/client/components/FeedbackMenu.js]: FIxed now

File: src/client/components/LectureTable.js (hunk 2 above)
Comment (teatimes) [reply src/client/components/LectureTable.js]: Used it to see where the wrong date came from. Removed now

**Review by weedle1912 — COMMENTED** [review]
The changes done under commit "Sending message when pushing enter" makes it possible to send empty message, as well as leaving a newline '\n' in the input field.

**Review by odgaard — APPROVED** [review]

**Pull request merged**